stream-chat-android-state/src/main/java/io/getstream/chat/android/state/plugin/state/internal/ChatClientStateCalls.kt

```diff
@@ -81,4 +81,23 @@ internal class ChatClientStateCalls(
         chatClient.getReplies(messageId, messageLimit).launch(scope)
         return state.thread(messageId)
     }
+
```

Comment:
Likewise has a non-conflicting change in it's `v6` thread PN navigation PR counterpart: https://github.com/GetStream/stream-chat-android/pull/4639/files#diff-56fdba7a308122f62f337bdda5c56fe732394b3674d4459686ca3d4177ea3780